usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -3179,7 +3199,11 @@ static CK_RV import_aes_xts_key(STDLL_TokData_t *tokdata, SESSION *sess,
     if (rc != CKR_OK)
         goto import_aes_xts_key_end;
 
+#ifndef NO_PKEY
     rc = ep11tok_pkey_check_aes_xts(tokdata, aes_xts_key_obj, CKM_AES_XTS);
+#else
+    rc = CKR_FUNCTION_NOT_SUPPORTED;
```

Comment:
Lets add a TRACE statement here in the else case that XTS is disabled due to no PKEY support.

Reply:
hm, well, the trace in the error handling below is clear enough.... So ignore this comment please.